server/pom.xml

```diff
@@ -43,6 +43,12 @@
             <artifactId>kubernetes-client</artifactId>
             <version>2.6.2</version>
         </dependency>
+        <!--Cloudformation plugin-->
```

Comment:
The base image mapper is still part of the K8s plugin. please consider putting your dependencies above the k8s plugin to keep it structured